pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
IMHO `{{path_to_base_dir}}` is pretty explicit, or maybe `{{prefix_path}}`?

It could help if we split the 2 examples, since I don't think both options are really related, e.g.
- start nginx with a specific config file (`-c` example)
- specify a prefix for all relative paths (`-p` example, which is nice since `-p` stands for `prefix`)